docs/compatibility.md

```diff
@@ -2,16 +2,23 @@
-| nodejs>=12, mjs                       |                                                                                           |
-| Browser using script tag, umd         |                                                                                           |
-| webpack@4                             |                                                                                           |
-| webpack@5                             |                                                                                           |
-| @vue/cli@4 (webpack@4)                |                                                                                           |
-| @vue/cli@5 (webpack@5)                |                                                                                           |
-| create-react-app@4 (webpack@4)        |                                                                                           |
-| create-react-app@5 (webpack@5)        |                                                                                           |
-| create-react-native-app@3 (webpack@4) | Requires `URLSearchParams` polyfill https://github.com/swagger-api/swagger-js/issues/2457 |
-| meteor@2                              |                                                                                           |
+| Environment                           | Comment                                                              |
+|---------------------------------------|----------------------------------------------------------------------|
+| nodejs>=12, common js                 |                                                                      |
+| nodejs>=12, mjs                       |                                                                      |
+| Browser using script tag, umd         |                                                                      |
+| webpack@4                             | requires a fix to work with mjs build [webpack-4]                    |
+| webpack@5                             |                                                                      |
+| @vue/cli@4 (webpack@4)                |                                                                      |
+| @vue/cli@5 (webpack@5)                |                                                                      |
+| create-react-app@4 (webpack@4)        | mjs build is not compatible with webpack@4 [cra-webpack-4]           |
+| create-react-app@5 (webpack@5)        |                                                                      |
+| create-react-native-app@3 (webpack@4) | mjs build is not compatible with webpack@4 [cra-webpack-4]           |
+| meteor@2                              |                                                                      |
+| jest@27.5.1                           | requires an environment where Buffer is instanceof Uint8Array [jest] |
```

Comment:
because of this check in tweetnacl: https://github.com/dchest/tweetnacl-js/blob/6a9594a35a27f9c723c5f1c107e376d1c65c23b3/nacl.js#L951-L952